### ISSUE TITLE
[Hue] Instruction for setup don't line up or work

### ISSUE THREAD

**Issue by peterfox** [raycast/extensions#8743]
### Extension

https://www.raycast.com/pindab0ter/hue

### Description

I can't setup the extension and it looks like the instructions are out of date.

https://github.com/raycast/extensions/tree/12f3cd668c0271bf4d016e454cfb50b4f8b0c872/extensions/hue/#setup

There is no `Manage Hue Bridge command`. When I try to run a command I see the option to set up, I click on the bridge but it fails with no errors. When I set the IP Address manually, the commands expect the button to be pressed still, I do this but receive an SSL Certificate error. See Screenshot.

<img width="862" alt="CleanShot 2023-10-09 at 19 33 45@2x" src="https://github.com/raycast/extensions/assets/1716506/888918e7-222a-4513-847e-bae7087358c6">

### Steps To Reproduce

1. Open Raycast
2. Run Control Lights
3. I see the click bridge pairing button
4. Hit enter
5. I see nothing

**Comment by pindab0ter** [comment]
The reason is that this is a state that shouldn't be reached. It is being reached by you and a handful of other users.

The API prescribes the use of SSL, so that's what I implemented. On older devices, this certificate is self-signed. I added support for this, but have no way to test it, as I don't own an old enough bridge.

To anyone reading this: I would love for someone with an old enough bridge to help me troubleshoot this issue.

Until then, there's not much I can do, unfortunately.
